.github/workflows/ci.yaml

```diff
@@ -93,6 +93,50 @@ jobs:
         env:
           CI_PACKED_CHARMS: ${{ needs.build.outputs.charms }}
 
+  integration-test-scaling:
```

Comment:
I actually think they were removed because they required 6 units of zookeeper, all requesting 10G memory, and the tests were flakey to run because of the high resource requirement.

I separated them from the other tests for that reason, as step in the job runs on it's own runner (in theory) with it's own resources.

If they're still flakey though, I might make them local-only. What do you think?

Reply:
sounds good!